redhat-cop/uncontained.io
# Migrating template guide; updating layouts to accomodate new articles

#### What is this PR About?
Migrating and revamping the template guide for the new site.

#### How should we test or review this PR?
Follow the instructions to [run the site preview](https://github.com/redhat-cop/uncontained.io/blob/master/CONTRIBUTING.md#building-and-developing-the-site), then:

- Read through the template guide content
- check that the article summaries look correct on the homepage, and articles pages
- Check the front matter

#### Is there a relevant Trello card or Github issue open for this?
n/a

#### Who would you like to review this?
cc: @redhat-cop/cant-contain-this

-----


-----

**Please note: we may close your PR without comment if you do not check the boxes above and provide ALL requested information.**
